package.json

```diff
@@ -56,6 +56,7 @@
     "@angular/compiler-cli": "^8.1.3",
     "@angular/language-service": "^8.1.3",
     "@babel/core": "^7.5.4",
+    "@ng-bootstrap/ng-bootstrap": "^5.1.0",
```

Comment:
@patrickf949, I noticed you added this library and you did not use its components, It's also good we keep consistent with a design approach that's being used by other members.